database/migrations/2025_02_20_104705_add_app_id_foreign_key_to_ugc_models.php

```diff
@@ -0,0 +1,44 @@
+<?php
```

Comment:
Temo che modificando i nomi delle migration si invalidi il comando vendor:publish --tag="wm-package-migrations" ... questo comando si accorge se una migration è già stata pubblicata e non la ripubblica. Qui vedo che hai mergiato delle migration assicurati che rilanciando il comando sopra non vengano create delle migration "doppie", in quel caso rimetti i nomi come quelli del wm-package